projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -356,20 +358,26 @@
 
 	.jetpack-option__type.jetpack-option__type {
 		align-items: center;
-		all: initial;
 		appearance: none;
-		color: var(--jetpack--contact-form--text-color);
 		display: flex;
-		border: none;
-		font-size: var(--jetpack--contact-form--font-size, 16px );
-		font-family: var(--jetpack--contact-form--font-family);
-		height: var(--jetpack--contact-form--font-size, 16px );
```

Comment:
This is a bit tricky - the inner input of an option needs to be able to inherit some of these styles from the parent block element, but these styles were overriding that. That was preventing the input from having the correct color and stopping it from scaling with smaller/bigger font sizes.

Where possible I've moved them to the parent block element (line 455, 456) so that this works again.